### PR TITLE
New post:Deploy an ASP.NET Core Application on Windows Server with AWS Lightsail

### PR DESCRIPTION
**Pull request by cobusbernard**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [ ] The title of this pull request reflects the title of my content.
- [ ] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [ ] This pull request contains one piece of content (post or page).

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
Please continue the review here, from old PR #350 . Due to making repo public.

File: posts/practical-cloud-guide/deploy-an-asp-net-core-application-on-windows-server-with-aws-lightsail/index.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+This is a brief overview of AWS Lightsail. As we progress through the tasks, we’ll go in depth with Lightsail’s other services.
+
+## Getting started
+
+In this tutorial you will create a Windows Server 2022 instance and deploy a ASP.NET Core application on IIS
+
+### Overview details table
+
+| Info                | Level                                  |
```
Comment (bketelsen) [inline posts/practical-cloud-guide/deploy-an-asp-net-core-application-on-windows-server-with-aws-lightsail/index.md:45]: needs to be changed

File: posts/practical-cloud-guide/deploy-an-asp-net-core-application-on-windows-server-with-aws-lightsail/index.md [hunk 2]
```diff
@@ -62,15 +63,15 @@ In this tutorial you will create a Windows Server 2022 instance and deploy a ASP
...
 
 In this module, the software and deployment script is in a GitHub repository. You will clone the repository to copy the files to your local drive. The next step is to create an S3 bucket to store the files that can be accessed by a Windows Server and deployed.
 
 ### Implementation Instructions
 
-Step 1: Clone the `practical-cloud-guide` repository. 
+Step 1: Clone the `practical-cloud-guide` repository.
 
 ```bash
 git clone <repo url>
```
Comment (spara) [inline posts/practical-cloud-guide/deploy-an-asp-net-core-application-on-windows-server-with-aws-lightsail/index.md:78]: git clone https://github.com/build-on-aws/practical-cloud-guide-code/

**Comment by cherbk** [comment]
new PR https://github.com/build-on-aws/content/pull/443

**Pull request closed without merging**